backend/controllers/admin-controller.js

```diff
@@ -9,7 +9,7 @@ const Patient = require("../models/patient");
  */
 async function updateRole(req, res) {
     if (req.body.newRole === req.body.oldRole) {
-        res.status(200);
+        await res.status(200).send("Same role");
```

Comment:
what does this change?

Reply:
I just put it to be safe with promises and stuff xd, shouldnt change anything except prevent any potential asynchronous problems in the future